deps/MPIconstants-1.4.0/.github/workflows/CI.yml

```diff
@@ -0,0 +1,164 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+env:
+  BUILD_TYPE: Debug
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os:
+          - ubuntu-20.04
+          - macos-11
+          #TODO - windows-2019
```

Comment:
Do we want to enable this as part of this PR?

Reply:
This isn't a CI file that is run for this project, this is merely a (complete) copy of the MPIconstants project.